examples/tilted_bottom_boundary_layer.jl

```diff
@@ -128,10 +128,12 @@ v_bcs = FieldBoundaryConditions(bottom = drag_bc_v)
 # and a constant viscosity and diffusivity.
 
 ν = 1e-4 # m² s⁻¹, small-ish
-κ = ν
-closure = ScalarDiffusivity(; ν, κ)
+closure = ScalarDiffusivity(ν=ν, κ=ν)
 
-model = NonhydrostaticModel(; grid, buoyancy, coriolis, closure,
+model = NonhydrostaticModel(; grid,
+                            buoyancy = buoyancy,
+                            closure = closure,
+                            coriolis = coriolis,
```

Comment:
I suggest we revert this...
I don't see what's the issue.
Anything after `;` is a `kwarg` and the order is not important.

Reply:
Yes, sorry I misread what was there and thought the ordered arguments version was being used. Thanks for catching that